04 - Guides, Workflows, & Courses/Guides/🗂️ Guides.md

```diff
@@ -21,6 +21,7 @@ tags:
 -  [[04 - Guides, Workflows, & Courses/Guides/How to add content through GitHub|How to add content through GitHub]]
 -  [[04 - Guides, Workflows, & Courses/Guides/How to add your plugin to the community plugin list|How to add your plugin to the community plugin list]]
 -  [[04 - Guides, Workflows, & Courses/Guides/How to add your theme to the community theme store|How to add your theme to the community theme store]]
+- [[How to debug why Obsidian is running slowly]]
```

Comment:
This isn't wrong, and doesn't need to be removed - but for future reference, the new line is in between a pair of comments like this, because the list of files is machine-generated by python script that gets run by a GitHub bot...

```text
%% Hub MOCs: Don’t edit below  %%
....
%% Hub MOCs: Don’t edit above  %%
```

> I wonder whether the following might be a good place to add a link to this new file?

> https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/for+Power+Users

Just to clarify that, the 'for Power Users' note is one of a number of notes that suggests notes will be useful for particular sorts of user, or particular areas....

Here are the ones we have so far, as copied for https://publish.obsidian.md/hub/04+-+Guides%2C+Workflows%2C+%26+Courses/🗂%EF%B8%8F+04+-+Guides%2C+Workflows%2C+%26+Courses

![image](https://user-images.githubusercontent.com/4840096/155815329-65c866bd-1822-48fb-92f8-f3fa590eb75a.png)

Looking at that list, I was kind of thinking that maybe it's Power Users that are more likely to be having trouble with their Obsidian Performance?

Reply:
yeah, I think having it in the "for Power Users" section makes sense.

Reply:
> Sorry, I'm not sure I follow. Did I mess up the formatting somewhere?

We have a script that generates these MOCs automatically, so this change will get overwritten, that is all.

Reply:
```md
%% Hub MOCs: Don’t edit below  %%
The links in here are created automatically and any changes here will get overridden.
%% Hub MOCs: Don’t edit above  %%